docs/en/00_Getting_Started/03_Environment_Management.md

```diff
@@ -83,8 +83,8 @@ SilverStripe core environment variables are listed here, though you're free to d
 | `SS_TRUSTED_PROXY_PROTOCOL_HEADER` | Used to define the proxy header to be used to determine HTTPS status |
 | `SS_TRUSTED_PROXY_IP_HEADER` | Used to define the proxy header to be used to determine request IPs |
 | `SS_TRUSTED_PROXY_HOST_HEADER` | Used to define the proxy header to be used to determine the requested host name |
-| `SS_TRUSTED_PROXY_IPS` | IP address or CIDR range to trust proxy headers from |
+| `SS_TRUSTED_PROXY_IPS` | IP address or CIDR range to trust proxy headers from. If left blank no proxy headers are trusted. Can be set to 'none' (trust none) or '*' (trust all) |
 | `SS_ALLOWED_HOSTS` | A comma deliminated list of hostnames the site is allowed to respond to |
 | `SS_MANIFESTCACHE` | The manifest cache to use (defaults to file based caching). Must be a CacheInterface or CacheFactory class name |
 | `SS_IGNORE_DOT_ENV` | If set the .env file will be ignored. This is good for live to mitigate any performance implications of loading the .env file |
-| `SS_HOST` | The hostname to use when it isn't determinable by other means (eg: for CLI commands) |
+| `SS_BASE_URL` | The url to use when it isn't determinable by other means (eg: for CLI commands). Protocol is optional. E.g. `//mysite.com/base-path` or `https://mysite.com/base-path` |
```

Comment:
The site could have an SSL and a non-ssl base url, in which case you may not want to fix it to one or the other.